pkg/scheduler/framework/v1alpha1/interface.go

```diff
@@ -150,10 +150,12 @@ func NewStatus(code Code, msg string) *Status {
 type WaitingPod interface {
 	// GetPod returns a reference to the waiting pod.
 	GetPod() *v1.Pod
-	// Allow the waiting pod to be scheduled. Returns true if the allow signal was
-	// successfully delivered, false otherwise.
-	Allow() bool
-	// Reject declares the waiting pod unschedulable. Returns true if the allow signal
+	// GetPendingPlugins returns a list of pending permit plugin's name.
+	GetPendingPlugins() []string
+	// Allow the waiting pod to be scheduled, it is allowed by plugin pluginName.
+	// Returns true if the allow signal was successfully dealt with, false otherwise.
```

Comment:
can you update the comment to say that "if this is the last remaining plugin to allow, then a success signal is delivered to unblock the pod"

Reply:
Updated. PTAL